### PR TITLE
Create modal-map

### PR DESCRIPTION
**Pull request by Ddemkowicz**

### CODE REVIEW

**Review by barteknr1 — CHANGES_REQUESTED** [review]
Twoja mapka jest widoczna na samym końcu i nie pojawia się po kliknięciu w przycisk, jest konflikt z index.html

File: src/js/modal-map.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/*(() => {
```
Comment (Grzybkowa) [inline src/js/modal-map.js]: It seems that file is not relevant anymore. Can you please remove it?

File: src/index.html [hunk 2]
```diff
@@ -42,6 +46,7 @@
     <include src="./partials/products-modal.html"></include>
     <!-- Skrypt jest na samym dole kodu. On juz ma w sobie podłączony SAAS plik. -->
     <script src="./js/index.js"></script>
+    <script src="./js/modal-map.js"></script>
```
Comment (Grzybkowa) [inline src/index.html]: As you're using open-modal.js, this one is not needed. Can you please remove this file as well? :)

File: src/js/modal-map.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  const refs = {
```
Comment (Grzybkowa) [inline src/js/modal-map.js]: Commented classes are not needed anymore. Can you please remove them ?

**Review by barteknr1 — APPROVED** [review]

**Pull request merged**